kube-app-testing.sh

```diff
@@ -995,23 +902,21 @@ print_help () {
   echo "                                  available types: kind, giantswarm"
   echo "  --cluster-name                  name of the cluster."
   echo "  -n, --namespace                 namespace to deploy the tested app to"
-  echo "  -a, --auth-token                auth token for the giantswarm API (only applies to"
-  echo "                                  giantswarm cluster type)"
   echo "  -r, --release-version           giantswarm release to use (only applies to"
   echo "                                  giantswarm cluster type)"
   echo "  --provider                      provider to deploy tenant cluster on"
   echo "                                  available providers: aws (default)"
   echo "  --availability-zone             availability zone to deploy the cluster into, defaults to"
   echo "                                  'eu-central-1a'"
-  echo "  --giantswarm-api-url            URL of the Giantswarm installation API, defaults to Gorilla."
-  echo "                                  e.g. 'https://api.g8s.gorilla.eu-central-1.aws.gigantic.io'"
   echo "  --min-scaling                   minimum number of nodes (applies to GS clusters only)"
   echo "  --max-scaling                   maximum number of nodes (applies to GS clusters only). If the max"
   echo "                                  value is set to _less_ than the min value then the provided max will"
   echo "                                  be ignored and max will be set to the same as min, resulting in a"
   echo "                                  statically-sized nodepool"
+  echo "  --no-external-kube-api          do not make GS clusters kubernetes api available from the internet"
+  echo "                                  (applies to GS clusters only)"
```

Comment:
i'm not sure of the point of this? if you don't do this then the cluster you spawn is useless - KAT will have no access to it to run the tests

Reply:
It works for me when I'm in VPN

Reply:
ok, so this flag is intended for when KAT is being run from your laptop? so we should just not use this flag when we're running in CircleCI?

Reply:
Exactly. Me and @webwurst will verify this still works in CI runs and make changes accordingly if this does not work as is :+1: